### ISSUE TITLE
Missing focus outline for checkboxes

### ISSUE THREAD

**Issue by dawidholisz** [Selleo/til#122]
Checkboxes in Add form don't have "focus" styles so when a user is using keyboard navigation, doesn't see that checkbox is focused
